tests/channels/test_d0_to_ks_kp_km.py

```diff
@@ -20,7 +20,7 @@ def test_script():
             "phi(1020)",
         ],
     )
-    stm.number_of_threads = 1
+    stm.number_of_threads = 2
```

Comment:
Why is this changed to 2 again? Afaik, this affects test coverage, see f99cfeab17f3449ae94a7e1af8220826182ecf9f

Reply:
Ah I forgot about the test coverage issue... I was just a remainder from pickling checks i did. I will change it back

Reply:
5e108b938402e1f7e0f229f1ee9b0a157c419be4